packages/atomic/src/pages/examples/ipx.html

```diff
@@ -225,19 +279,24 @@
           <atomic-layout-section section="pagination">
             <atomic-load-more-results class="pagination-more-result"></atomic-load-more-results>
           </atomic-layout-section>
+          <atomic-recs-interface fields-to-include='["cat_rating_count"]'>
+            <atomic-recs-list label="Recommended articles" number-of-recommendations="10"> </atomic-recs-list>
```

Comment:
Regardless of whether or not you apply my other comments, I definitely think we should have a result template here. The default is more of a dev placeholder IMO.